src/transac/components/components.module.coffee

```diff
@@ -0,0 +1,10 @@
+###
+#   @desc The Components module is the container reference for all reusable modules. Modules added as depenedencies here should be decoupled.
+###
+angular.module('transac.components',
```

Comment:
I debated with myself on this one a fair bit. As I have coded Transac! in a way which each module could be removed from the project and run given the parent component's required bindings are provided.

If I put all the modules under `transac`, then it doesn't really indicate what is stand alone. But this is also mainly following todd motto's example apps and the way he seperates the angular modules. What do you think?